west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
       revision: 12d7647870888e4cb0e421f2b26884c2e76915ac
       path: modules/hal/nordic
     - name: hal_openisa
-      revision: 9bfbe35aad79fdf552dbdae47a894e26be7c022c
+      revision: pull/4/head
```

Comment:
refering to head of a pull request?

Reply:
Exactly. So hal_openisa PR will have to be merged before this one, and updated to a sha instead.

Reply:
it was